index.js

```diff
@@ -1 +1,93 @@
 // implement your API here
+const express = require('express');
+
+const db = require('./data/db.js');
+
+const server = express();
+
+server.use(express.json());
+
+server.get('/', (req, res) => {
+    res.send("Server setup in progress...");
+});
+
+server.post('/api/users', (req, res) => {
+    const newUser = req.body;
+    if (!name || !bio) {
```

Comment:
right now name and bio are meaningless to the code, we didn't define those anywhere, newuser.name however, will be recognized, swap these out and it'll be workin jus fine